internal/proto/codec/client_fetch_schema_codec.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Can you link the protocol PR associated with these codecs?

Reply:
There's no protocol PR, since https://github.com/hazelcast/hazelcast-client-protocol/pull/420 is WIP.

Reply:
They are auto-generated, but some manual changes are necessary, like the imports.

Reply:
OK, then since this is not something specific to Compact, I am resolving this